macros/src/lib.rs

```diff
@@ -790,6 +830,30 @@ fn extract_cfgs(attrs: Vec<Attribute>) -> (Vec<Attribute>, Vec<Attribute>) {
     (cfgs, not_cfgs)
 }
 
+fn check_for_blacklisted_attrs(attrs: &[Attribute]) -> Option<TokenStream> {
+    if let Some(val) = containts_blacklist_attrs(attrs) {
+        return Some(parse::Error::new(val.span(), "This attribute is not allowed [blacklisted]")
```

Comment:
```suggestion
        return Some(parse::Error::new(val.span(), "this attribute is not allowed on a function controlled by cortex-m-rt")
```

For extra points, mention the exact type of cortex-m-rt attribute here (exception handler, interrupt handler, entry point).